pkgs/top-level/python-packages.nix

```diff
@@ -3624,6 +3624,8 @@ in {
 
   minimock = callPackage ../development/python-modules/minimock { };
 
+  swspotify = callPackage ../development/python-modules/swspotify { };
+
```

Comment:
since the time this PR was opened, python-packages was sorted, please attempt to place this in the correct location.